dask/dataframe/core.py

```diff
@@ -5878,6 +5878,7 @@ def apply_concat_apply(
         chunk,
         *args,
         token=chunk_name,
+        meta=dfs[0],
```

Comment:
I do think a comment here that the meta isn't real, it's just to avoid inference, would be helpful